libgalois/include/katana/PropertyGraph.h

```diff
@@ -431,6 +431,14 @@ class KATANA_EXPORT PropertyGraph {
     return edge_entity_type_id_.data();
   }
 
+  EntityTypeManager GetCopyOfNodeTypeManager() const {
+    return EntityTypeManager(node_entity_type_manager_);
+  }
+
+  EntityTypeManager GetCopyOfEdgeTypeManager() const {
```

Comment:
Instead, why don't we return a const-reference to the member variable and let the user do a copy of it?

Reply:
That works for me